src/DataViews/FunctionsDataView.cpp

```diff
@@ -36,8 +36,11 @@ FunctionsDataView::FunctionsDataView(AppInterface* app,
     : DataView(DataViewType::kFunctions, app, metrics_uploader) {}
 
 const std::string FunctionsDataView::kUnselectedFunctionString = "";
-const std::string FunctionsDataView::kSelectedFunctionString = "✓";
+const std::string FunctionsDataView::kSelectedFunctionString = "H";
 const std::string FunctionsDataView::kFrameTrackString = "F";
+const std::string FunctionsDataView::kApiScopeType = "MS";
+const std::string FunctionsDataView::kApiScopeAsyncType = "MA";
+const std::string FunctionsDataView::kHookedFunctionType = "D";
```

Comment:
Just personal preference, but I would like to use a different name here for the current one is a bit confusing:
We use "H" to indicate selected / hooked functions and so I might misunderstand that kHookedFunctionType is for all hooked function.

Reply:
It's `kDynamicallyInstrumentedFunctionTypeString` now.